backend/runtimes/runtimes.go

```diff
@@ -1,18 +1,85 @@
 package runtimes
 
 import (
+	"go.autokitteh.dev/autokitteh/internal/backend/configset"
+	"go.autokitteh.dev/autokitteh/internal/backend/httpsvc"
 	"go.autokitteh.dev/autokitteh/internal/kittehs"
 	configruntimesvc "go.autokitteh.dev/autokitteh/runtimes/configrt/runtimesvc"
 	pythonruntime "go.autokitteh.dev/autokitteh/runtimes/pythonrt"
 	starlarkruntimesvc "go.autokitteh.dev/autokitteh/runtimes/starlarkrt/runtimesvc"
 	"go.autokitteh.dev/autokitteh/sdk/sdkruntimes"
 	"go.autokitteh.dev/autokitteh/sdk/sdkservices"
+	"go.uber.org/zap"
 )
 
-func New() sdkservices.Runtimes {
-	return kittehs.Must1(sdkruntimes.New([]*sdkruntimes.Runtime{
+type Config struct {
+	RemoteRunnerEndpoints []string `koanf:"remote_runner_endpoints"`
+	RemoteRunner          bool     `koanf:"enable_remote_runner"`
+	WorkerAddress         string   `koanf:"worker_address"`
+	// TODO: This is a hack to prevent running configure on pythonrt in each test
+	// which currently install venv everytime and takes a really long time
+	// need to find a way to share the venv once for all tests
+	PythonRTEnabled bool `koanf:"enable_python_runtime"`
```

Comment:
Try to fix this with `TestMain` and `AK_WORKER_PYTHON`

Reply:
refactored, irrelevant